### ISSUE TITLE
maxWaitTimeBeforePassingControlToApp Times Out Behind Load Balancer With SSL

### ISSUE THREAD

**Issue by treyrich2001** [balderdashy/skipper#48]
Hey So I wanted to open dialog about this, I can submit a pull request if a decision is reached on how this should be resolved. But what's happening is when my app is running behind an AWS load balancer with SSL as the protocol the maxWaitTimeBeforePassingControlToApp is taking place before the app has the ability to process the textparams. If I increase this variable a bit everything works great.

I'm not sure if there was any research done on how long this should be or if the 50 was arbitrary, but I think the most simple solution would be to create a config variable that influences this. I'm not sure how that could be done as skipper is really sails so could this be done in the application config safely? What are other alternatives?

**Comment by mikermcneil** [comment]
@treyrich2001 Hey Trey, thanks for the heads up.  This should be resolved in 0.5.5 if you wouldn't mind giving that a go?  Easiest way is to go in your sails project and run:

```
rm -rf node_modules/sails
npm cache clear
npm install sails@beta --save --force
# this will install the latest skipper as well
```
_(alternatively, you could clone this repo, run `npm link` in there, then run `npm link skipper` in your sails projects' sails dep, i.e.  `cd ./node_modules/sails && npm link skipper`)_

**Comment by treyrich** [comment]
@mikermcneil sorry to take so long to circle back to this, I ended up just making my own custom skipper with a longer `maxWaitTimeBeforePassingControlToApp` to work around this last year so it hasn't been an issue again until now.

We're facing the exact same problem as before where the `maxWaitTimeBeforePassingControlToApp` is expiring before the first textparam arrives. On the client side we're using AFNetworking in an iOS app. Increasing the `maxWaitTimeBeforePassingControlToApp` fixes the issue instantly, but coming back to my original question is there any way without forking the repo to modify this? It says in the comments of the code that it's not modifiable on purpose, but I can't figure out what that purpose might be?

I'm happy to put together a pull request to fix this issue, but I'm not completely sure what the best course of action is to solve it for everyone? Maybe it's only an issue with AFNetworking?
